.travis.yml

```diff
@@ -0,0 +1,55 @@
+sudo: required
+
+language: go
+
+services:
+  - docker
+
+addons:
+  apt:
+    packages:
+      - docker-ce
+
+os:
+  - linux
+
+go:
+  - "1.12"
+
+env:
+  global:
+    - BUILD_VERSION=$(echo ${TRAVIS_COMMIT} | cut -c 1-10)
+    - MAIN_GO_VERSION=1.12
+    - GORACE="halt_on_error=1"
+
+install:
+  - make deps
+
+script:
+  - make test;
+  - if [[ "$TRAVIS_GO_VERSION" == "$MAIN_GO_VERSION" && "$TRAVIS_OS_NAME" == "linux" ]]; then
+    release_latest=$( [[ "$TRAVIS_BRANCH" == "master" && "$TRAVIS_TAG" == "" && "$TRAVIS_PULL_REQUEST_BRANCH" == "" ]] && echo "yes" || echo "no" );
+    release_tag=$( [[ "$TRAVIS_TAG" != "" ]] && echo "yes" || echo "no" );
+
+    if [[ $release_latest == "yes" ]]; then
+    docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD";
+    BUILD_VERSION="latest" make docker
+    docker logout;
+    fi;
+
+    if [[ $release_tag == "yes" ]]; then
+    docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD";
+    BUILD_VERSION="$TRAVIS_TAG" make docker
```

Comment:
please note that you have have the `v` in the tag name.

Reply:
good catch!

Reply:
I think I will remove the v from the Makefile to make it easier